### ISSUE TITLE
Missing \Throwable on ExceptionInterface

### ISSUE THREAD

**Issue by FabianKoestring** [LM-Commons/LmcRbacMvc#139]
### Bug Report

| Q          | A     |
|------------|-------|
| Version(s) | 2.0.0 |

#### Summary
We use phpstan to check our codebase. With the newest version we get phpstan errors like ....

```
Class Lmc\Rbac\Mvc\Exception\UnauthorizedException does not have a constructor and must be instantiated without any parameters.
```
and 
```
Invalid type Lmc\Rbac\Mvc\Exception\UnauthorizedException to throw.
```

The code we use, is the code from the https://lm-commons.github.io/LmcRbacMvc/docs/Guides/example docs.
```php
public function deletePost($id)
{
    if (!$this->authorizationService->isGranted('deletePost')) {
        throw new UnauthorizedException('You are not allowed !');
    }
}
```

#### Expected behavior

I looked at the exception classes in `LmcRbac` and `LmcRbacMvc`. Shouldnt the class https://github.com/LM-Commons/LmcRbac/blob/2.0.x/src/Exception/ExceptionInterface.php extend `\Throwable`??

**Comment by visto9259** [comment]
@FabianKoestring 
Something's not right.  
The class was modified from the previous version and there was no reason to, except for the namespace change.  It should be extending RuntimeException.  I don't recall why I did that and obviously, the test suite is not catching this error.
Let me look into it.
BTW this is a LmcRbacMvc issue so I will move it to LmcRbacMvc's issue list.

**Comment by visto9259** [comment]
A stupid oversight during refactoring.
`LmcRbacMvc\Exception\UnauthorizedException` was extending RuntimeException.
During the refactoring, I somehow removed the extension.
Now `Lmc\Rbac\Mvc\Exception\UnauthorizedException` is now extending `Lmc\Rbac\Exception\RuntimeException` which ultimately extends `\Exception` and is therefore throwable.
This was not caught because LmcRbacMvc never throws this exception. But if the user's code does, it would not work.
Thanks @FabianKoestring for reporting this issue.
Fixed in 4.0.1

**Comment by FabianKoestring** [comment]
Gladly. Thank you for your prompt implementation.
